com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -561,15 +612,22 @@ static void OnAdditionalProbesBakeCompleted()
                     SphericalHarmonicsL2Utils.SetCoefficient(ref cell.sh[i], 7, new Vector3(shv[0, 7], shv[1, 7], shv[2, 7]));
                     SphericalHarmonicsL2Utils.SetCoefficient(ref cell.sh[i], 8, new Vector3(shv[0, 8], shv[1, 8], shv[2, 8]));
 
-                    cell.validity[i] = validity[j];
+                    float currValidity = invalidatedProbe ? 1.0f : validity[j];
```

Comment:
maybe a dumb question but why do you pur the validity to 1 if the probe is invalid ?

Reply:
Because it's the other way round (yeah not something I really like, but that's the data that comes from the lightmapper)

Reply:
then when you check 'validity[j] < 0.05f' it's true if probe is valid ?

Reply:
yes, we invalidate only valid probes via the touchup volumes (those are the only one whose validity for the sake of occlusion weight computation can be restored)